.github/workflows/haskell.yml

```diff
@@ -94,6 +94,19 @@ jobs:
       run: cabal v2-build --enable-tests --enable-benchmarks all
     - name: Run all tests (cabal)
       run: cabal v2-test all
+    - name: Run doctest (docspec)
+      run: |
+        # Install docspec and run doctests.
+        echo "=========================[ INSTALL DOCSPEC ]========================"
+        echo " -> Started at $(date --iso-8601=seconds --utc)."
+        curl -sL https://github.com/phadej/cabal-extras/releases/download/cabal-docspec-0.0.0.20211114/cabal-docspec-0.0.0.20211114.xz > ./cabal-docspec.xz
+        xz -d < ./cabal-docspec.xz > ./cabal-docspec
+        rm -f ./cabal-docspec.xz
+        chmod a+x ./cabal-docspec
+        echo " -> Docspec has been installed."
+        echo "===========================[ RUN DOCSPEC ]==========================="
+        ./cabal-docspec
```

Comment:
is there no need to mention `cabal build libs` before this - did it already happen because of some previous `run`?

Reply:
The haskell workflow defined in this file includes [quite a few different steps](https://github.com/geniusyield/atlas/actions/runs/4561069855/jobs/8046620186).

Before running `Run doctest (docspec)` the `Build all targets (cabal)` is run:

![image](https://user-images.githubusercontent.com/2914096/228752432-5194e36b-2a50-49e0-884a-6feea9689cb2.png)

This is running the following command:

```
cabal v2-build --enable-tests --enable-benchmarks all
```

which is building everything.